core/src/main/java/com/airwallex/airskiff/flink/udx/StdDevAccumulator.java

```diff
@@ -0,0 +1,7 @@
+package com.airwallex.airskiff.flink.udx;
+
+import org.apache.flink.table.api.dataview.ListView;
+
+public class StdDevAccumulator {
```

Comment:
you can keep an avg or (sum and total_cnt) in this accumulator as a state, by exposing add/remove functions, instead of exposing ListView directly.

Reply:
got it